presto-docs/src/main/sphinx/connector/jmx.rst

```diff
@@ -81,6 +81,23 @@ for each node::
                          329 |                  10240
     (1 row)
 
+In ``current`` schema it is allowed to use wildcard ``*`` character in table name.
+That way it is possible to match several MBean objects with single query.
+Below query return information of available memory for different Presto
+memory pools on each node::
+
+    SELECT freebytes, node, object_name
+    FROM jmx.current."com.facebook.presto.memory:*type=memorypool*";
+
+.. code-block:: none
+
+     freebytes  |                 node                 |                       object_name
+    ------------+--------------------------------------+----------------------------------------------------------
+      214748364 | ffffffff-ffff-ffff-ffff-ffffffffffff | com.facebook.presto.memory:type=MemoryPool,name=reserved
```

Comment:
How about shorting the node ID to `example`? That allows the table to be narrower and makes it less distracting.